Tests/Common/IRBase58Tests.m

```diff
@@ -7,8 +7,11 @@
 //
 
 @import XCTest;
-@import NovaCrypto;
+#import "NSData+Hex.h"
 #import "IRBase58TestData+Load.h"
+#if __has_include(<NovaCrypto/NSData+Base58.h>)
```

Comment:
Could you, please, remind why we can import `NSData+Hex.h` but not `NSData+Base58.h`